src/components/ScenarioParameters/components/ScenarioParametersInputs/GenericEnumInput.js

```diff
@@ -101,6 +101,12 @@ export const GenericEnumInput = ({
     return rawEnumValues;
   }, [t, parameterData, dynamicEnumValues]);
 
+  useEffect(() => {
+    if (parameterValue == null && enumValues.length > 0) {
+      setParameterValue(enumValues?.[0]?.key);
```

Comment:
If _enumValues._length_ > 0_ there is no need to put optional chaining to `enumValues?.[0]`
On the other side, maybe put optional chaining to `enumValues?.length` ?